lib/crawlers/company_crawler/company_list_crawler.rb

```diff
@@ -0,0 +1,44 @@
+require_relative 'company_crawler'
+require 'pathname'
+require 'fileutils'
+require 'csv'
+
+module Crawlers
+  class CompanyListCrawler
```

Comment:
We should make a rake task to be able to call this easily

Reply:
Only vaguely aware of rake tasks at this point, will look at examples

Reply:
Done